### PR TITLE
Improve readability of docs by using code tag [ci skip]

### PR DESCRIPTION
**Pull request by amitsuroliya**

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/object/inclusion.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class Object
-  # Returns true if this object is included in the argument. Argument must be
+  # Returns +true+ if this object is included in the argument. Argument must be
```
Comment (kaspth) [inline activesupport/lib/active_support/core_ext/object/inclusion.rb]: Actually we'd only want to mark true like that if the return value is `true` and not just truthy. So revert this change.

**Comment by amitsuroliya** [comment]
@kaspth PR updated. Thanks for review :heart:

**Comment by kaspth** [comment]
Thanks!
